lib/search.rb

```diff
@@ -7,11 +7,23 @@ def self.create(environment = Rails.env)
     config_path = Rails.root + 'config' + "elasticsearch.yml"
     client_config = HashWithIndifferentAccess.new(YAML.load_file(config_path))
     client_config = client_config[environment].merge(client_config[:all_envs])
+    index_name = client_config.delete(:index)
```

Comment:
Oh, I see - because we pass this config into the client below. I find this a little surprising. Wouldn't it be a bit cleaner to divide the config up into things that are forwarded on to the client, and things that aren't?